plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/GlobalConfigurationCategoryConfigurator.java

```diff
@@ -72,10 +74,19 @@ public Set describe() {
                 .filter(d -> d.getCategory() == category)
                 .filter(d -> d.getGlobalConfigPage() != null)
                 .map(d -> new DescriptorConfigurator(d))
+                .filter(GlobalConfigurationCategoryConfigurator::reportDescriptorWithoutSetters)
                 .map(c -> new Attribute<GlobalConfigurationCategory, Object>(c.getName(), c.getTarget()).setter(NOP))
                 .collect(Collectors.toSet());
     }
 
+    public static boolean reportDescriptorWithoutSetters(Configurator c) {
+        if (c.describe().isEmpty()) {
+            logger.warning(c.getTarget().getName() +
+                    " has a global view but CasC didn't detected any configurable attribute; see: https://jenkins.io/redirect/casc/");
```

Comment:
Seems sensible to me and not at all rude

Reply:
@ndeloof 
Let's use a more verbose redirect.  It seems like `redirect/casc-attributes` would make sense.